frog/imports/client/TeacherView/SessionUtils.jsx

```diff
@@ -177,7 +177,7 @@ const SessionUtils = ({
         </Typography>
       </Grid>
       <Grid item xs={4} className={classes.textCenter}>
-        <DashToggle visible={visible} toggleVisible={toggle} />
+        {!session.singleActivity && <DashToggle visible={visible} toggleVisible={toggle} />}
```

Comment:
Because session is of type any, we have no guarantee that singleActivity would exist. If within the scope of this PR, we could add a prop type for this component specifying the type of the session object

Reply:
With reference to my previous comment, I think no change is required here.